lib/blocks.php

```diff
@@ -76,6 +76,7 @@ function gutenberg_reregister_core_block_types() {
 					'post-excerpt.php'              => 'core/post-excerpt',
 					'post-featured-image.php'       => 'core/post-featured-image',
 					'post-hierarchical-terms.php'   => 'core/post-hierarchical-terms',
+					'post-navigation-link.php'      => 'core/post-navigation-link',
```

Comment:
Would `post-previous-next` be more clear?

Reply:
I'm not really sure about that.. I've no strong opinion on this though.